public/app/features/integrated-alerting/components/NotificationChannel/AddNotificationChannelModal/PagerDutyFields/PagerDutyFields.tsx

```diff
@@ -3,16 +3,28 @@ import { TextInputField, validators } from '@percona/platform-core';
 import { Messages } from '../AddNotificationChannelModal.messages';
 import { PagerDutyFieldsProps } from './PagerDutyFields.types';
 
-export const PagerDutyFields: FC<PagerDutyFieldsProps> = ({ values }) => {
-  const { required } = validators;
-  const { routing, service } = values;
-  const routingValidators = service ? [] : [required];
-  const serviceValidators = routing ? [] : [required];
+/**
+ * This is a validator factory.
+ * Given a key, return a validator that will check if there's anything there.
+ */
+const anotherKeyRequired = (complementaryKey: string): validators.Validator => (
+  value: any,
+  allValues: Record<string, any>
+): undefined | string => {
```

Comment:
FYI & nit: `undefined | string` should be part of the `Validator` type

Reply:
👍